sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -341,6 +342,25 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
+  test("describe partition") {
+    withTable("partitioned_table") {
+      sql("CREATE TABLE partitioned_table (a STRING, b INT) PARTITIONED BY (c STRING, d STRING)")
+      sql("ALTER TABLE partitioned_table ADD PARTITION (c='Us', d=1)")
+
+      sql("DESC partitioned_table PARTITION (c='Us', d=1)")
```

Comment:
Actually, as you see in the PR description, the output result is the same with the `DESC` command with `PARTITION` spec.  In other words, the output does not depend on the given spec. Only, this command raises exceptions if the given partition does not exists.

> the output result points out the partition columns correctly.